Resources/Prototypes/Construction/power.yml

```diff
@@ -36,8 +36,8 @@
     amount: 2
 
 - type: construction
-  name: APC
-  id: apc
+  name: apc
```

Comment:
This is an acronym, right? So it should be `APC`.

Reply:
It is yeah.